test/TestHelper.cpp

```diff
@@ -172,6 +201,74 @@ void ImportTest::importTransaction(json_spirit::mObject& _o)
 	}
 }
 
+void ImportTest::checkExpectedState(State const& _stateExpect, State const& _statePost, WhenError _throw)
```

Comment:
there is stateOptionsMap const& in second constructor. which requires an instance of stateOptionsMap. 

it could be changed to stateOptionsMap const = stateOptionsMap(), but thus it would not be transfered via &